src/installation/live-images/partitions.md

```diff
@@ -40,7 +43,7 @@ following table has recommendations for swap partition size.
 
 | System RAM | Recommended swap space | Swap space if using hibernation |
 |------------|------------------------|---------------------------------|
-| < 2GB      | 2x the amount of RAM   | 3x the amount of RAM            |
+| &lt; 2GB   | 2x the amount of RAM   | 3x the amount of RAM            |
```

Comment:
Is this necessary for some of the other stuff you all have going on? Otherwise, We'd like to keep the markdown as human readable as possible.

Reply:
Ah. It was causing some syntax highlighting issues with my editor. Should I change it back?

Reply:
Please do c:

Reply:
Will do! Sorry about that.

Reply:
No worries ;)